packages/wonder-blocks-form/src/components/labeled-text-field.md

```diff
@@ -333,57 +333,6 @@ const styles = StyleSheet.create({
 <LabeledTextFieldExample />
 ```
 
-The field can have custom styles
```

Comment:
I wouldn't remove this test here, and would update it with an example of how custom styles can be applied in the container, so devs can understand possible uses. (same would apply to the story).

One good example could be the "two fields in a row" that seems to be a real case that devs could benefit from!